frontend/next.config.mjs

```diff
@@ -291,6 +291,7 @@ export default withTM({
         return [
             {source: '/nieuws', destination: '/actueel'},
             {source: '/inzendingen', destination: '/actueel'},
+            {source: '/petities/:slug', destination: '/post/:slug'}
```

Comment:
Kan je hier enkelfout van maken? Dus /petitie. Net zoals bij /post, wat niet /posts is